taf/APL/APL_Runner.py

```diff
@@ -14,14 +14,16 @@ def __init__(self,
                  state_code: str,
                  run_id: str,
                  job_id: int,
-                 file_version: str):
+                 file_version: str,
+                 run_stats_only: bool = False):
         super().__init__(da_schema,
                          reporting_period,
                          state_code,
                          run_id,
                          job_id,
                          file_version)
 
+        self.run_stats_only = run_stats_only
```

Comment:
The notebook is passing run_stats_only as a string, which is always resolving to True later on when doing `if run_stats_only`. I believe this variable needs to be explicitly casted to a bool everywhere.

Reply:
Hmmm. I have the code here with run_stats_only: bool = False. Are you sure you didn't look at an older checkin? I had to make a couple of updates for the reason you stated.

Reply:
BTW: You can't even cast.

Reply:
```
>>> bool(int('0'))
False
>>> bool(int('1'))
True
```
This might work better if we specify 0/1 in the parameter.